src/pivot-table/components/cells/ColumnAdjuster.tsx

```diff
@@ -21,14 +22,18 @@ interface AdjusterProps {
  * While dragging this components follows the pointer, and on mouse up all column widths are updated.
  */
 const ColumnAdjuster = ({ cellInfo, columnWidth, dataModel, isLastColumn }: AdjusterProps) => {
+  const { isActive } = useSelectionsContext();
   const [internalWidth, setInternalWidth] = useState(columnWidth);
   const tempWidth = useRef({ initWidth: 0, columnWidth: 0, initX: 0 });
   const positionAdjustment = isLastColumn ? CELL_PADDING : CELL_PADDING + GRID_BORDER;
+  const shouldRender = !isActive && cellInfo.canBeResized;
```

Comment:
Should also include `interactions.active` in this check. So that you cannot re-size in edit mode in Sense.

Reply:
yeah there is a fix PR comming up next, didn't want to mix it with this refactor